src/system.jl

```diff
@@ -0,0 +1,302 @@
+"""
+    ServicesTimeSeries(reg, spin, sup_on, sup_off)
+
+Type defining services time series.  Fields are KeyedArray where the keys are generator names
+x datetimes.  Only generators that provide each service are included in the array.
+"""
+struct ServicesTimeSeries
+    reg::KeyedArray{Float64}
+    spin::KeyedArray{Float64}
+    sup_on::KeyedArray{Float64}
+    sup_off::KeyedArray{Float64}
```

Comment:
are these all matrices?
```suggestion
    sup_off::KeyedArray{Float64, 2}
```

Reply:
Yes they are